doc/man7/flux-broker-attributes.rst

```diff
@@ -34,22 +37,86 @@ rundir
    directory, it is removed.  In most cases this attribute should not
    be set by users.
 
-content.backing-path
-   The path to the content backing store file(s). If this is set on the
-   broker command line, the backing store uses this path instead of
-   a temporary one, and content is preserved on instance exit.
-   If file exists, its content is imported into the instance.
-   If it doesn't exist, it is created.
+security.owner
+   The numeric userid of the owner of this Flux instance.
+
+local-uri
+   The Flux URI that should be passed to :man3:`flux_open` to
+   establish a connection to the local broker rank.
+
+parent-uri
+   The Flux URI that should be passed to :man3:`flux_open` to
+   establish a connection to the enclosing instance.
+
+instance-level
+   The nesting level of this Flux instance (zero if there is no enclosing
+   Flux instance).
+
+jobid
+   The Flux jobid of this Flux instance, if it was launched by Flux as a job.
+
+parent-kvs-namespace
+   The KVS namespace assigned to this Flux instance, if it was launched by
+   Flux as a job.
 
 hostlist
    An RFC 29 hostlist in broker rank order.
 
+broker.mapping
+   A string representing the mapping of broker ranks to nodes,
+   in the Argonne format described in RFC 13.
+
+broker.pid
+   The process id of the local broker.
+
+broker.quorum
+   The ranks that are required to be online before the rank 0 broker enters
+   the RUN state and starts the initial program (if any).  Normally this is
+   all ranks, except on a system instance.
+
+broker.quorum-timeout
+   The amount of time (in RFC 23 Flux Standard Duration format) that the
+   rank 0 broker waits for the ``broker.quorum`` set to come online before
+   aborting the Flux instance.
+
+broker.rc1_path
+   The path to the broker's rc1 script.
+
+broker.rc3_path
+   The path to the broker's rc3 script.
+
 broker.starttime
    Timestamp of broker startup from :man3:`flux_reactor_now`.
 
+conf.connector_path
+   A colon-separated list of directories that are searched by :man3:`flux_open`
+   to resolve the scheme component of a URI.
```

Comment:
This might be confusing since the attribute does not actually influence `flux_open(3)` but is instead inherited from the `FLUX_CONNECTOR_PATH` environment variable (and I'm not sure anything is even done with it after that). Therefore it is in some ways informational only.

Similar comment to the other `conf.*` attributes (except `pmi_library_path` I think), though `conf.exec_path` is used by `flux(1)`, but only when used with the `--parent` option.

Also, I note that these attributes can be set. Maybe a separate issue would be whether that should be possible.

Reply:
Good point - I'll update the documentation to reflect that.

Just opened #4120 about making them immutable since that probably should be a separate release notes item.

Reply:
I ended up auditing all of the entries to add defaults, and annotations showing whether each attribute can be updated on the command line or at runtime.

Reply:
Nice!

Reply:
I noticed a readthedocs rendering issue in the RESOURCE section so if it's OK I'd like to squash the fixup and force push.

Reply:
Yeah, sounds good

Reply:
OK, pushed.